### ISSUE TITLE
Shanghai Teng Yuan Logistics Co video data

### ISSUE THREAD

**Issue by yaoyuanww** [filecoin-project/notary-governance#862]
## Application information 

Link to existing LDN application(s)
- _link to applocation(s) in https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1939
  
Are you the data owner?
- [√ ] Yes
- [ ] No

Is the data private i.e., encrypted or not meant to be retrieved by anyone else on the network?
- [√ ] Yes
- [ ] No

## Data owner information
- Data owner: Shanghai Teng Yuan Logistics Co
- Data owner's website: https://www.gkeml.com/
- Data owner's social media account(s):

### Data owner description 
Shanghai Tengyuan Logistics Co., Ltd., established in 2015 with a registered capital of RMB 2 million, is located in Pudong New Area, Shanghai. We are a professional logistics company that integrates car transportation and cargo transportation. Our predecessor was "Siyuan Logistics". After 20 years of development and operation, we have formed a complete logistics distribution network. Our company owns more than 30 large, medium, and small professional transport vehicles, forming a professional transport team with abundant assets, advanced transportation equipment, and excellent vehicle conditions. At the same time, we have a large warehouse base with an area of approximately 20,000 square meters, including 11,000 square meters of indoor space and 9,000 square meters of outdoor space. With strong strength, fast transportation, efficient focus, and safe logistics services, Shanghai Tengyuan Logistics Co., Ltd. has been highly praised and recommended by customers.

Relying on the resource advantages of our partner GKE (Singapore) Metal Logistics Company, in addition to our own warehousing, logistics, and customer resources, we also share resources with our partner in designated LME warehouses in Singapore, bonded warehouses in Shanghai Waigaoqiao Free Trade Zone and Tianjin Free Trade Zone, etc.

### What is the motivation behind onboarding this data to Filecoin? 
We are planning to store massive amounts of historical video surveillance data from multiple warehouses on the Filecoin network. Firstly, this will prepare us for the financial institution to call up goods monitoring videos at any time. Secondly, these video surveillance data will also provide a dataset for AI machine learning.

Filecoin is a decentralized storage network based on blockchain technology, which can provide reliable data storage and access services. Storing historical video data on the Filecoin network can provide high data security and reliability. In addition, by using the Filecoin network, we can fully utilize its distributed storage features and store data in multiple locations, thereby reducing the risk of data loss.

### What organizations are involved in this project? Sources of funding, SPs partnering in onboarding, other service providers, etc.?
Those warehouse video surveillance data is extremely important for the business, to have those data stored in a decentralized, safe and cost effective way is something we are looking for a long time, Filecoin network comes highly recommended to us in such capacity.
We are working closely with Linkspeed to get onboard with Filecoin network, and we will fund this project on our own.

## Transparency in KYC
_If you are not the data owner, please email filplus@fil.org with you and the data owner on a thread so at least the Fil+ governance team can act as a lightweight KYC check._

Are you going to offer NDAs to notaries?
- [ ] Yes
- [ √] No
  
## Dataset description 
We are planning to store massive amounts of historical video surveillance data from multiple warehouses on the Filecoin network. Firstly, this will prepare us for the financial institution to call up goods monitoring videos at any time. Secondly, these video surveillance data will also provide a dataset for AI machine learning.

### Why is it useful to the Filecoin network?
Filecoin is a decentralized storage network based on blockchain technology, which can provide reliable data storage and access services. Storing historical video data on the Filecoin network can provide high data security and reliability. In addition, by using the Filecoin network, we can fully utilize its distributed storage features and store data in multiple locations, thereby reducing the risk of data loss.
  
## Storage needs

Is the data to be stored in a single region, either due to geopolitical constraints or data owner policy?
- [ ] Yes
- [√ ] No

### Which SPs will you/the data owner be working with to store the data? 
_If all the SPs have not been chosen at the time of issue submission, feel free to edit and update the form as they are chosen. If you need support on finding reputable SPs, please check out filgram.io, filrep.io, plus.fil.org/miners, or Filecoin Slack channels: #fil-plus, #fil-deal-market_
  
| SP ID | SP org | SP region |
| f01847751 | --- | USA |
| f02058333 |Linkspped | USA |
| f01059489 | Linkspped | USA |
| _minerID_ | _org_ | _region_ |

### How were these SPs identified?
we are working with Linkspeed, for the rest SPs, we will find them through referral or slack

## Notaries that support the project 
 _Notaries perform KYC/KYB to validate the legitimacy of data and organizations. To find supporting notaries please view the list of [round 3 notaries](https://github.com/filecoin-project/notary-governance/tree/main/notaries) and post your LDN application in the Filecoin slack channel #fil-plus-application-review. Feel free to update the form as you gain support over time and more notaries are willing to support your application._

| Notary name | Notary org | Notary region |
| --- | --- | --- |
| _name_ | _org_ | _region_ |

**Comment by liyunzhi-666** [comment]
I don't think this application should appear here, @yaoyuanww you should submit the issue here https://github.com/filecoin-project/filecoin-plus-large-datasets/issues.
cc @Kevin-FF-USA ,can you delete or close this issue?
